# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/frontend/ParseNode.h
# Commit: 1b8c42577e24
# Full Hash: 1b8c42577e2410dbbe050bb30b0ccced9d7c3771
# Author: Ashley Hauck <khyperia@mozilla.com>
# Date: 2019-03-20 10:52:37
# Regressor Bug: 1535166
# File Overlap Count: 1
# Description:
#   Bug 1535166 - Implement computed field names. r=jorendorff
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D23408
# ==============================================================================

diff -r 28370bf14b97 -r 1b8c42577e24 js/src/frontend/ParseNode.h
--- a/js/src/frontend/ParseNode.h	Tue Mar 19 22:02:49 2019 +0000
+++ b/js/src/frontend/ParseNode.h	Tue Mar 19 20:23:03 2019 +0000
@@ -1780,9 +1780,9 @@
   }
 };
 
-// This is only used internally, currently just for tagged templates.
-// It represents the value 'undefined' (aka `void 0`), like NullLiteral
-// represents the value 'null'.
+// This is only used internally, currently just for tagged templates and the
+// initial value of fields without initializers. It represents the value
+// 'undefined' (aka `void 0`), like NullLiteral represents the value 'null'.
 class RawUndefinedLiteral : public NullaryNode {
  public:
   explicit RawUndefinedLiteral(const TokenPos& pos)